core-tests/src/test/java/org/mule/runtime/core/processor/strategy/ProactorProcessingStrategyTestCase.java

```diff
@@ -63,8 +66,9 @@ public void singleCpuLight() throws Exception {
     super.singleCpuLight();
     assertThat(threads.size(), equalTo(1));
     assertThat(threads.stream().filter(name -> name.startsWith(CPU_LIGHT)).count(), equalTo(1l));
```

Comment:
change asserts. sames as the line below, but without the `not`. Check other occurrences